### DISCUSSION TITLE
Aalen Additive Fitter

### DISCUSSION THREAD

**Discussion by camferna** [CamDavidsonPilon/lifelines#1357] [Q&A] [answered]
Hello,

What is the method used to estimate the parameters of Aalen Additive Fitter? Is it by minimizing the least square error?

Thank you and regards,

Camila F

**Comment by CamDavidsonPilon** [comment, marked as answer]
Hi @CamilaFernandez8, it uses linear regression, which is a least-squares techniques, so yes.
